src/sdk/pynni/nni/tuner.py

```diff
@@ -93,3 +93,16 @@ def _on_exit(self):
 
     def _on_error(self):
         pass
+
+    def extract_scalar_reward(self, value, scalar_key='default'):
+        if isinstance(value, float) or isinstance(value, int):
```

Comment:
ok, we still support float and int, right?

Reply:
Yes, I just moved those logic from dispatcher to tuner, to ensure that all the previous examples & trial apis won't be changed.